tests/test_validate_metadata.py

```diff
@@ -103,15 +119,13 @@ def test_valid_nonontology_content(self):
     def test_invalid_nonontology_content(self):
         """Non-ontology metadata should conform to convention requirements
             """
-        args = "../tests/data/AMC_v0.8.json " "../tests/data/metadata_invalid.tsv"
+        args = "../tests/data/AMC_v1.1.00.json " "../tests/data/metadata_invalid.tsv"
```

Comment:
The double 0 padding in `AMC_v1.1.00.json` is somewhat unconventional.  [Semantic Versioning](https://semver.org) uses unpadded integers for each of the major, minor and patch versions -- e.g. `AMC_v1.1.0.json` would be more conventional versioning.

Reply:
updated in `4023a08`